client/index.html

```diff
@@ -90,12 +89,14 @@
   <script src="/bower_components/moment/moment.js"></script>
   <script src="/bower_components/moment-timezone/builds/moment-timezone-with-data-2010-2020.js"></script>
   <script src="/bower_components/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js"></script>
+  <script src="/bower_components/patternfly-bootstrap-treeview/dist/bootstrap-treeview.min.js"></script>
   <script src="/bower_components/patternfly/dist/js/patternfly.js"></script>
   <script src="/bower_components/angular-patternfly/dist/angular-patternfly.js"></script>
   <script src="/bower_components/angular-drag-and-drop-lists/angular-drag-and-drop-lists.js"></script>
   <script src="/bower_components/angular-resource/angular-resource.js"></script>
   <script src="/bower_components/angular-svg-base/src/svgDirs.js"></script>
   <script src="/bower_components/angular-ui-router/release/angular-ui-router.js"></script>
+  <script src="/bower_components/bootstrap-treeview/dist/bootstrap-treeview.min.js"></script>
```

Comment:
Should we be depending on both bootstrap-treeview **and** patternfly-bootstrap-treeview? Seems like one is a fork of the other, right?